bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/Client.scala

```diff
@@ -108,15 +133,22 @@ trait Client
   def getDaemon: BitcoindInstance = instance
 
   override lazy val cmd: String = {
-    val binaryPath = instance.binary.getAbsolutePath
-    val cmd = List(binaryPath,
-                   "-datadir=" + instance.datadir,
-                   "-rpcport=" + instance.rpcUri.getPort,
-                   "-port=" + instance.uri.getPort)
-    logger.debug(
-      s"starting bitcoind with datadir ${instance.datadir} and binary path $binaryPath")
-
-    cmd.mkString(" ")
+    instance match {
+      case _: BitcoindInstanceRemote =>
+        sys.error(
+          s"Cannot start remote instance with local binary command. You've likely misconfigured something")
```

Comment:
imo it'd be better to just use an empty string here and have `startBinary` do nothing when it is a `BitcoindInstanceRemote`

Reply:
I think ben is right, can you just replace the `sys.error()` -> `logger.warn()`